VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -275,13 +275,12 @@ public void onSuccess(AuthResponse result) {
                             }
                         } catch (LoginException ex) {
                             super.onException(ex);
+                            onUserLoginFailure(ex, null, null);
                         }
                     }
-
                     @Override
                     public void onException(Exception ex) {
                         super.onException(ex);
-                        onUserLoginFailure(ex, null, null);
```

Comment:
Now that I look at this, I don't know if this was ever was called. Looks like the getAuthResponse exception went right to the showErrorMessage()